core/templates/dev/head/pages/splash.html

```diff
@@ -8,30 +8,114 @@
   Oppia
 {% endblock subtitle %}
 
+{% block header_js %}
+  {{ super() }}
+
+  <style>
+    html, body {
+      background-color: #AFD2EB;
+    }
+  </style>
+{% endblock %}
+
 {% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+    <li>
+      <!--
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      Splash
+      -->
+    </li>
+  </ul>
 {% endblock navbar_breadcrumb %}
 
 {% block content %}
-  <div class="oppia-gallery-container">
-    <div class="oppia-gallery-banner-container" style="background-color: #888;">
-      <div class="oppia-gallery-banner-content" style="height: 100%; width: 100%;">
-        Hi, I&rsquo;m Oppia.
-        <br>
-        <span style="font-size: 0.8em;">
-          This splash page needs to be replaced.
-        </span>
-      </div>
+
+  <div id="oppia-splash-section-one" class="text-center">
```

Comment:
Can you use a class here, rather than an id? We do that in general for CSS, since ids are supposed to be just one-per-page and we don't really have a good way of enforcing that in general.